### PR TITLE
added isNullOrEmpty check for firstname and lastname

### LINKED ISSUE #38: null null is displayed in profile if name and last name is not set
screenshot_20180531-082202

working on it 👍

### PR DESCRIPTION
**Pull request by simarsingh24**
fixes #38

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt [hunk 1]
```diff
@@ -65,7 +65,8 @@ class ProfileFragment : Fragment() {
                 }
                 .subscribe({ user ->
                     Timber.d("Response Success")
-                    rootView.name.text = "${user.firstName} ${user.lastName}"
+                    if (!(user.firstName.isNullOrEmpty() && user.lastName.isNullOrEmpty()))
+                        rootView.name.text = "${user.firstName} ${user.lastName}"
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: A user may have first name and no last name, and vice versa

Do this

"${firstname.nullToEmpty()} ${lastname.nullToEmpty()"

Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: for null strings, this would mean "true true". Not sure this will work

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: nullToEmpty != isNullOrEmpty

**Comment by iamareebjamal** [comment]
Resolve Conflicts

**Comment by simarsingh24** [comment]
@iamareebjamal  please merge this

**Pull request merged**